sources/packages/backend/workflow/src/workflow-definitions/load-assessment-consolidated-data.bpmn

```diff
@@ -109,6 +109,7 @@
           <zeebe:header key="studentDataStudentParentsCppLine30800" value="$.data.studentParentsCppLine30800" />
           <zeebe:header key="studentDataStudentParentsCppLine31000" value="$.data.studentParentsCppLine31000" />
           <zeebe:header key="studentDataStudentParentsEiLine31200" value="$.data.studentParentsEiLine31200" />
+          <zeebe:header key="studentDataStudentCurrentYearParentIncome" value="$.data.parents.currentYearParentIncome" />
```

Comment:
`$.data.parents` is an array. Hence it does not have a property by name `currentYearParentIncome`. 

We instead should add the additional property mapped into the array here. 

<img width="638" height="183" alt="image" src="https://github.com/user-attachments/assets/676a0cca-3045-420e-a585-b44581982f22" />

<img width="653" height="166" alt="image" src="https://github.com/user-attachments/assets/53d431d7-7c95-43a9-971c-f54643bb32c3" />